packages/ppxx/ppxx.1.2.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+version: "1.2.1"
+authors: "Jun Furuse"
+maintainer: "jun.furuse@gmail.com"
+bug-reports: "jun.furuse@gmail.com"
+dev-repo: "hg://https://bitbucket.org/camlspotter/ppxx"
+build: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-build" ]
+]
+install: [
+  [ "ocaml" "setup.ml" "-install" ]
+]
+remove: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind"
+  "omake"
```

Comment:
If these are only used in the build process and not linked, they can be marked `build` to help reduce the build tool upgrade recompile-the-world burden.